app/views/coronavirus_landing_page/components/landing_page/_page_header.html.erb

```diff
@@ -17,7 +16,6 @@
       </div>
     </div>
   </div>
-
```

Comment:
is there any reason why these spaces have been removed?

Reply:
updated due to `stylelint-gds`